u3-views/features/courseController.js

```diff
@@ -0,0 +1,52 @@
+const { getCourse } = require('../_services/fakedb')
+
+const courseDetails = (req, res) => {
+  const courseCode = req.params.code
+  const course = getCourse(courseCode)
+
+  // left `coure` = neme in handlebars
+  // right `coure` = JS variable
+
+  res.render('course-details', {
```

Comment:
พยายามหลีกเลี่ยงช่องว่างระหว่างความคิดเห็นและรหัสที่กำลังพูดถึง!
Try to avoid spaces between a comment and the code that it is discussing!